lib/pbench/test/unit/server/test_quisby_results.py

```diff
@@ -0,0 +1,116 @@
+from http import HTTPStatus
+from pathlib import Path
+
+from pquisby.lib.post_processing import QuisbyProcessing
+import pytest
+import requests
+
+from pbench.server.api.resources import ApiBase
+from pbench.server.cache_manager import CacheManager
+from pbench.server.database.models.datasets import Dataset, DatasetNotFound
+
+
+class TestQuisbyResults:
+    @pytest.fixture()
+    def query_get_as(self, client, server_config, more_datasets, get_token_func):
+        """
+        Helper fixture to perform the API query and validate an expected
+        return status.
+
+        Args:
+            client: Flask test API client fixture
+            server_config: Pbench config fixture
+            more_datasets: Dataset construction fixture
+            pbench_drb_token: Authenticated user token fixture
```

Comment:
should it be `get_token_func`?

Reply:
My guess is that the mismatch was copied from somewhere else that wasn't properly updated with the Keycloak conversion. For bonus points, you could fix that also. 😆